serverless/cmd/sequence/main.go

```diff
@@ -26,12 +26,13 @@ import (
 	"path/filepath"
 
 	"github.com/google/trillian-examples/formats/log"
-	"github.com/google/trillian-examples/serverless/internal/storage"
 	"github.com/google/trillian-examples/serverless/internal/storage/fs"
 	"golang.org/x/mod/sumdb/note"
 
 	"github.com/golang/glog"
 	"github.com/transparency-dev/merkle/rfc6962"
+
+	stErrors "github.com/google/trillian-examples/serverless/pkg/storage/errors"
```

Comment:
Did you consider moving it to `...pkg/log`?
I wonder if returning this concrete error should be part of the documented `Storage` interface there?

Reply:
Good point. Done.